src/libraries/adaptive-curve/ConstantsLib.sol

```diff
@@ -15,11 +19,11 @@ library ConstantsLib {
     int256 public constant TARGET_UTILIZATION = 0.9 ether;
 
     /// @notice Initial rate at target per second (scaled by WAD).
-    int256 public constant INITIAL_RATE_AT_TARGET = int256(0.04 ether) / 365 days;
+    int256 public constant INITIAL_RATE_AT_TARGET = 4_00 * ONE_BPS / 365 days;
```

Comment:
That's the point ^^ To have the reviewer naturally read `4.00` to indicate it corresponds to 4%

Reply:
I don't like it neither. What was the issue with 0.04 ether?

Reply:
It is less obvious that it corresponds to 4%

Reply:
Probably most readers seeing `4_00` will be either misled (by thinking the actual literal value is not `400` but `4`, when what they should understand is "`4_00` is a hint that the entire `400 * ONE_BPS` product means 4% because the value is in wads and 0.04 is 4%") or confused. The goal of spelling out "4%" more clearly is good though, but writing it clearly in a comment with be direct and simple.

Reply:
I added a hint that the initial rate corresponds to a 1% min APR / 16% max APR

Reply:
sorry but for me 0.04 ether is still simpler and clearer, I propose this
```suggestion
    /// @dev Initial rate at target is 4%.
    int256 public constant INITIAL_RATE_AT_TARGET = 0.04 ether / 365 days;
```